tool_collections/bcftools/macros.xml

```diff
@@ -0,0 +1,83 @@
+<macros>
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.1093/bioinformatics/btp352</citation>
+        </citations>
+    </xml>
+    <xml name="filter_conditional">
+        <conditional name="filter_calls">
+            <param label="Filter by regions" name="filter_calls_selector" type="select">
+                <option selected="True" value="no_filter">Do not filter regions</option>
+                <option value="regions">Filter by regions (-R)</option>
+                <option value="samples">Filter samples (-S)</option>
+                <option value="targets">Filter targets (-T)</option>
+            </param>
+            <when value="no_filter" />
+            <when value="regions">
+                <param format="bed,vcf,tabular" label="File listing regions to include" name="input_regions" type="data" />
+            </when>
+            <when value="samples">
+                <param label="Output ploidy" name="output_ploidy" type="select">
+                    <option selected="True" value="">Diploid output</option>
+                    <option value="-X">Haploid output for male samples (-X)</option>
+                    <option value="-Y">Haploid output for male samples, skip female (-Y)</option>
+                </param>
+                <param format="ped" label="PED file with regions to include" name="input_regions" type="data" />
+            </when>
+            <when value="targets">
+                <param format="bed,vcf,tabular" label="File listing regions to include" name="input_regions" type="data" />
+            </when>
```

Comment:
Something I noticed from the docs today:

> Both -r and -t options can be applied simultaneously

these aren't actually mutually exclusive options :)